Extensions/Content/Dnn.PersonaBar.Extensions/WebApps/Pages.Web/src/actions/addPagesActions.js

```diff
@@ -37,6 +37,13 @@ const addPagesActions = {
         };
     },
 
+    addPagesComplete() {
+        return (dispatch) => {
+            dispatch({
+                type: ActionTypes.UNFORCE_PAGE_LIST_LOAD
+        })};
```

Comment:
Missing semicolon after closing parenthesis ")":
```
    addPagesComplete() {
        return (dispatch) => {
            dispatch({ type: ActionTypes.UNFORCE_PAGE_LIST_LOAD });
        };
    },
```

Reply:
Updated as suggested, thank you.